server/lib/config/pbench-server-default.cfg

```diff
@@ -128,17 +128,37 @@ lowerbound = 820
 # uri = driver://user:password@hostname/dbname
 wait_timeout = 120
 
+[authentication]
+# Secret key for use by the Pbench Server API sub-system for security related
+# tasks.
+secret-key = my_precious
+
+# User authentication section to use when authorizing the user with an OIDC
+# identity provider.
+
+# URL of the OIDC auth server
+#server_url = http://keycloak-broker.example.com:8080
+
+# Realm name that is used for the authentication with OIDC
+realm = pbench
+
+# Client entity name requesting OIDC to authenticate a user
+client = pbench-client
+
+# Client secret if the above client is not public
+keycloak-secret = <keycloak_secret>
+
+[apache]
 # We need to install some stuff in the apache document root so we
 # either get it directly or look in the config file.
 #
 # N.B. Different distros use different config files.  The following
 # works on Fedora, RHEL, CentOS.
-[apache]
 documentroot = /var/www/html
 configfile = /etc/httpd/conf/httpd.conf
 
-# This *has* to agree with the setting in the pbench-agent config file.
 [results]
+# This *has* to agree with the setting in the pbench-agent config file.
 host-info-path-prefix = pbench-results-host-info.versioned/pbench-results-host-info.URL
 host_info_url = http://%(default-host)s/%(host-info-path-prefix)s
```

Comment:
Since we don't support the whole `ssh` protocol, we should probably remove these lines entirely.

Reply:
Removed in PR #3081.